pkg/controllers/gateway/gateway_controller.go

```diff
@@ -271,6 +272,12 @@ func (r *GatewayReconciler) reconcileGateway(ctx context.Context, previous gatew
 			}
 		}
 	}
+
+	//set jsonpatch annotation
+	metadata.AddAnnotation(gateway, "mctc-syncer-patch/kind-mctc-workload-1", "[\n"+
```

Comment:
We don't want this hardcoded to the `kind-mctc-workload-1` cluster.
Right now, the syncObjectToAllClusters func below uses a special annotation to sync the object to all clusters. 
I've reached out to Phil to see if there's an equivalent for the patching annotation.
If not, we can hold this PR, pending #128. The list of clusters can be fetched from the annotations added by the placement controller when that lands